### PR TITLE
Port bootloader to actuator-board

### PR DESCRIPTION
**Pull request by antoinealb**
Mostly based on can-io-board:

* Changed LED pin
* Removed CAN shutdown pin and do not enable GPIOB.

### CODE REVIEW

File: platform/actuator-board/include.jinja [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+{% for dir in include_directories %}
+PACKAGER_INCDIR += {{ dir }}
+{% endfor %}
+
+{% for file in target['armv7-m'] + target.stm32f3 + source %}
+{% if file.endswith('.c') %}CSRC += {{ file }}
+{% elif file.endswith('.s') %}ASMSRC += {{ file }}
+{% elif file.endswith('.cpp') %}CXXSRC += {{ file }}
+{% endif %}{% endfor %}
```
Comment (SyrianSpock) [inline platform/actuator-board/include.jinja:10]: would an explicit makefile make sense? Do we want to continue using packager?

**Review by SyrianSpock — APPROVED** [review]

File: platform/actuator-board/include.jinja (hunk 1 above)
Comment (antoinealb) [reply platform/actuator-board/include.jinja:10]: Do we want to keep using packager? No
Is migrating can-bootloader a priority? Nope :)

Comment (SyrianSpock) [reply platform/actuator-board/include.jinja:10]: OK, fair enough

**Review by msplr — APPROVED** [review]
Looks good to me.
Do you want to merge now or wait until we can test on hardware?

**Comment by antoinealb** [comment]
yes, merging it now.

**Pull request merged**